shared/admin/index.html

```diff
@@ -160,7 +160,7 @@ <h3 class="panel-title">Connection Settings</h3>
     <!-- /.container -->
     <div id="footer">
       <div class="container">
-          <p class="text-muted text-right credit"><b>InfluxDB</b> v0.9.3</p>
+          <p class="text-muted text-right credit"><b>InfluxDB</b> <span class="influxdb-version"></span></p>
```

Comment:
I think id can be used here instead of class to match another tag lookups in javascript.